library/ctypes.po

```diff
@@ -7,34 +7,44 @@ msgstr ""
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2019-12-05 23:16+0100\n"
 "PO-Revision-Date: 2018-07-29 19:07+0200\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: FRENCH <traductions@lists.afpy.org>\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: Antoine Wecxsteen\n"
+"X-Generator: Poedit 2.2.1\n"
 
 #: ../Doc/library/ctypes.rst:2
 msgid ":mod:`ctypes` --- A foreign function library for Python"
-msgstr ""
+msgstr ":mod:`ctypes` — Bibliothèque Python d'appels à des fonctions externes"
 
 #: ../Doc/library/ctypes.rst:11
 msgid ""
 ":mod:`ctypes` is a foreign function library for Python.  It provides C "
 "compatible data types, and allows calling functions in DLLs or shared "
 "libraries.  It can be used to wrap these libraries in pure Python."
 msgstr ""
+":mod:`ctypes` est une bibliothèque d'appel à des fonctions externes en "
+"python. Elle fournit des types de données compatibles avec le langage C et "
+"permet d'appeler des fonctions depuis des DLLs ou des bibliothèques "
```

Comment:
```suggestion
"permet d'appeler des fonctions depuis des DLL ou des bibliothèques "
```
Les sigles sont invariables en français.

Reply:
Noté.